src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -591,8 +591,7 @@ def load_command_table(self, _):
         g.custom_command('list-shared', 'sig_shared_image_definition_list')
 
     vm_shared_gallery_image_version = CliCommandType(
-        operations_tmpl='azure.mgmt.compute.operations._shared_gallery_image_versions_operations#SharedGalleryImageVers'
-        'ionsOperations.{}',
+        operations_tmpl='azure.mgmt.compute.operations#SharedGalleryImageVersionsOperations.{}',
```

Comment:
Does the SDK also break the operation path?

Reply:
it seems the path is also changed: https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-compute_30.0.0/sdk/compute/azure-mgmt-compute/azure/mgmt/compute/v2022_01_03/operations/_operations.py#L6089